src/EntityFramework6.Npgsql/SqlGenerators/VisitedExpression.cs

```diff
@@ -844,6 +844,8 @@ internal enum UnaryTypes {
         public static readonly Operator QueryNegate = new Operator("!!", 10, 8, UnaryTypes.Prefix, true);
         public static readonly Operator QueryContains = new Operator("@>", 10, 8);
         public static readonly Operator QueryIsContained = new Operator("<@", 10, 8);
+        public static readonly Operator RegexMatch = new Operator("~*", 10, 8);
+        public static readonly Operator RegexMatchCase = new Operator("~", 10, 8);
 
```

Comment:
Please rename RegexMatchCase to just RegexMatch and remove the insensitive one (it is unused).